rotkehlchen/tests/unit/decoders/test_cowswap.py

```diff
@@ -1012,8 +1038,21 @@ def test_gnosis_eure_v2(
         asset=Asset('eip155:100/erc20:0x420CA0f9B9b604cE0fd9C18EF134C705e5Fa3430'),
         balance=Balance(FVal(received_amount)),
         location_label=user_address,
-        notes=f'Receive {received_amount} EURe as the result of a swap in cowswap',
+        notes=f'Receive {received_amount} EURe as the result of a cowswap market order swap',
         tx_hash=tx_hash,
         counterparty=CPT_COWSWAP,
         address=string_to_evm_address('0x9008D19f58AAbD9eD0D60971565AA8510560ab41'),
+    ), EvmEvent(
```

Comment:
I don't see any limit order tests. We need one. Don't we have any test after 19/03/2024? According to the issue after that everything is a limit order

Reply:
I've added a limit order test, but note that actually the existing `test_swap_token_to_eth_with_other_trade` test is after 19/03/2024, but it is a market order (there are still market orders, the change was only in how they are executed internally).